assets/xml/objects/object_fsn.xml

```diff
@@ -53,7 +55,8 @@
         <!-- <Blob Name="object_fsn_Blob_00FB30" Size="0x3040" Offset="0xFB30" /> -->
         <Animation Name="object_fsn_Anim_012C34" Offset="0x12C34" />
         <Animation Name="object_fsn_Anim_0131FC" Offset="0x131FC" />
-        <!-- <Blob Name="object_fsn_Blob_01320C" Size="0x124" Offset="0x1320C" /> -->
+        <LimbTable Name="object_fsn_LimbTable_0132DC" LimbType="Standard" Count="17" Offset="0x132DC"/>
+        <Skeleton Name="object_fsn_Skel_013320" Type="Flex" LimbType="Standard" Offset="0x13320" />
```

Comment:
Can you add a bug comment that the header defines 18 limbs, even though there are only 17?